tools/check_format.py

```diff
@@ -12,6 +12,8 @@
 import stat
 import sys
 import traceback
+import shutil
```

Comment:
Nit: can you remove shutil from this and `envoy_build_fixer.py`? Good to go otherwise, thanks!
/wait

Reply:
done